modules/internal/String.chpl

```diff
@@ -736,30 +921,55 @@ module String {
       return codepointToString(this.codepoint(idx));
     }
 
+    /*
+      Index into a string
```

Comment:
I think "Index into a string" is a jargon way of putting it.

I might just say "Return the i'th codepoint in a string".